src/utils.py

```diff
@@ -86,7 +86,7 @@ def process_tpf_query_text(resp, raw_repo_uri, call_name, extraMetadata):
         pagination_param['type'] = "int"
         pagination_param['in'] = "query"
         pagination_param['description'] = "The page number for this paginated query ({} results per page)".format(pagination)
-        params.append(pagination_param)
+        # params.append(pagination_param) # Does this get used elsewhere?
```

Comment:
Oh, good catch. This is because for TPFs pagination is actually the only parameter possible. Which means right now pagination is not working for TPF queries :) If you can initialize params = [] before 'if pagination:' that should correct it.